.github/ISSUE_TEMPLATE/branch-checklist.md

```diff
@@ -21,10 +21,10 @@ _Descriptions of these steps can be found in the team OneNote._
 - [ ] For branches not matching `dev*` or `feature/*` (usually we skip these)
   - [ ] Update the [pull-request.yml](https://github.com/dotnet/project-system/blob/main/eng/pipelines/pull-request.yml) (via `pr`) to support PR builds
   - [ ] Update the [official.yml](https://github.com/dotnet/project-system/blob/main/eng/pipelines/official.yml) (via `trigger`) to have signed builds the new branch
-- [ ] Update Roslyn Tools [config.xml](https://github.com/dotnet/roslyn-tools/blob/main/src/GitHubCreateMergePRs/config.xml) file to flow branch changes to the latest dev branch
-  - [ ] dotnet/roslyn-tools PR: https://github.com/dotnet/roslyn-tools/pull/❓
```

Comment:
I guess we can clean up a bunch of old config from this file too.

Reply:
Yeah, I need to do that before I update our dev17.9.x branch to use the correct installer channel; otherwise we're going to try and merge that back to main.